lessons/bn/basics/collections.md

```diff
@@ -79,14 +82,16 @@ iex> tail
 [:pie, "Apple"]
 ```
 
-## টাপ্ল 
+## টাপ্ল
```

Comment:
I think `টাপল` is better.

Reply:
yes, just found the original pull request #969 where this was introduced. it was to be fixed later. making the fix. thanks.

Reply:
fixed.